net/fail2ban/Makefile

```
@@ -75,6 +77,11 @@ define Py3Package/fail2ban/install
 	$(INSTALL_BIN) ./files/fail2ban.defaults $(1)/etc/uci-defaults/99_fail2ban
 endef
 
+define Py3Build/Compile
+	$(SHELL) -c "cd $(PKG_BUILD_DIR)/$(PYTHON3_PKG_SETUP_DIR) && ./fail2ban-2to3";
```

```suggestion
	cd $(PKG_BUILD_DIR) && ./fail2ban-2to3
```

`$(PYTHON3_PKG_SETUP_DIR)` is only needed if your package had set `PYTHON3_PKG_SETUP_DIR` earlier.

---

Okay.
Thanks for the `PYTHON3_PKG_SETUP_DIR` coorection (and for the explanation also ;-) ).
I think I have already tested something like this, with direct cd command instead of $SHELL call.
`	cd $(PKG_BUILD_DIR) && ./fail2ban-2to3`
I will test with your modification, and if it do not work, I revert with your modification like:
` 	$(SHELL) -c "cd $(PKG_BUILD_DIR) && ./fail2ban-2to3";`

---

okay with your proposal, another issue I get before, looks to be fixed now, but I still do not understand why !